.dependabot/config.yml

```diff
@@ -0,0 +1,10 @@
+version: 1
```

Comment:
does this need to be in our packages specifically?

Reply:
```
The config file must start with version: 1 followed by an array of update_configs.
```

Reply:
No. It can be at the root and watch all nested packages.

Reply:
oh this is the language server, duh